src/util/errors-ts.ts

```diff
@@ -1027,3 +1027,25 @@ export class LambdaSizeExceededError extends NowError<
     });
   }
 }
+
+export class MissingDotenvVarsError extends NowError<
+  'MISSING_DOTENV_VARS',
+  { type: string, missing: string[] }
+> {
+  constructor(type: string, missing: string[]) {
+    let message: string;
+    if (missing.length === 1) {
+      message = `Env var ${JSON.stringify(missing[0])} is not defined in \`${type}\``;
```

Comment:
Maybe we could use `src/utils/output/code` to format this `${type}`?

Reply:
Good call. Updated in 456969ca.